packages/dev/core/src/Misc/textureTools.ts

```diff
@@ -244,6 +241,14 @@ const ProcessAsync = async (texture: BaseTexture, width: number, height: number,
     const scene = texture.getScene()!;
     const engine = scene.getEngine();
 
+    if (!engine.isWebGPU) {
+        if (texture.isCube) {
+            await import("../Shaders/lodCube.fragment");
+        } else {
+            await import("../Shaders/lod.fragment");
+        }
+    }

```

Comment:
We probably need to import the WGSL versions when the engine is WebGPU [...] These shaders have not been ported to WGSL yet, so maybe add a log or throw an exception when in WebGPU mode in the meantime?